tools/lkl/lib/hijack/hijack.c

```diff
@@ -283,3 +283,20 @@ int epoll_ctl(int epollfd, int op, int fd, struct epoll_event *event)
 
 	return lkl_call(__lkl__NR_epoll_ctl, 4, epollfd, op, fd, event);
 }
+
+int eventfd(unsigned int count, int flags)
+{
+	return lkl_sys_eventfd2(count, flags);
```

Comment:
@pscollins you're right. ```eventfd``` will never touch the host.

```epoll_create``` also has the same issue: no way to distinguish the destination without additional information. I currently have no idea how to deal with it.


> This is a useful discussion but it doesn't seem to apply here.

right . my bad.
my comment only applies to ```eventfd_read``` and ```wirte```.